src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/jira/DataCenterFormula.kt

```diff
@@ -281,14 +350,15 @@ class DataCenterFormula private constructor(
             address = loadBalancer.uri,
             jmxClients = jiraNodes.mapIndexed { i, node -> configs[i].remoteJmx.getClient(node.publicIpAddress) }
         )
-        logger.info("$jira is set up, will expire ${jiraStack.expiry}")
+        logger.info("$jira is set up, will expire ${jiraStack.expiry}. External access to the application is restricted with firewall (See JPERF-730).")
```

Comment:
Issue key might not say much to non-Atlassian users. How about having an URL to the tickets instead?

Reply:
It's a common practice for the log messages to contain issue keys formatted the Jira way. At least for me if I search in google "JPERF JPT" it gives me some results directly leading to the EAN issues, so I believe this should be fine unless the issue will need to remain closed.

I will consider creating another one as a placeholder to make it public.

Reply:
Even if it is a common practice, then an actual URL is a more practical practice :wink:

Reply:
I'm not sure about spamming log with URLs. I tried to make the message as short as possible.

Probably we should remove it at some point. I don't know when though. Any suggestions are welcome.

Reply:
- [ ] Add JPERF-730 URL to the "Jira is set up" log message
- [ ] Note in the code when the JPERF-730 URL can be removed from the log message

Reply:
@dagguh, @mzyromski-atlassian, @mgrzaslewicz I'm not sure if I should add the issue key / URL to the "Jira is set up" log message at all. The rationale behind it was to mention it together with the Jira's URL, so that if user is using it they could notice that something changed.

If the issue key should be kept in this log message I'm wondering how long we should keep it? Should it be forever there or can we assume that after some time consumers will understand the new behavior? Any suggestions from you?

Reply:
I'm on the fence here. I think we can even drop the issue link altogether assuming we'll have a proper, detailed description of the change in the changelog. I don't have a strong opinion, so whatever you decide @pczuj will be fine for me.